solution4.py.py

```diff
@@ -0,0 +1,14 @@
+# writing salary csv file
+count=0
+import csv
+with open("salary.csv",'r',newline='') as f:
+    data= csv.writer(f)

```

Comment:
can i write this statement like,
data=writer(f)

it is valid or not ? If valid then How

Reply:
yes

Reply:
yes from import csv  used for data=writer(f)